pom.xml

```diff
@@ -110,8 +110,10 @@
     <profile>
       <id>run-code-coverage</id>
       <properties>
-        <jacoco.exec.file>${project.build.directory}/jacoco.exec</jacoco.exec.file>
         <jacoco.excludes>*Lexer:org.optaplanner.core.impl.testdata.domain.*:*TestdataAbstractSolutionBasedSolution</jacoco.excludes>
+
+        <!-- jacoco.exec.file is defined in kie-parent -->
+        <!--suppress UnresolvedMavenProperty -->
```

Comment:
code style nitpick (good to merge): Normally we have a space after <!-- or //

Reply:
will fix that